src/components/PriceLine.jsx

```diff
@@ -33,11 +33,12 @@ class PriceLine extends Component {
         const showBarrier = !(hideOffscreenBarrier && offScreen);
         const showBarrierDragLine = !hideBarrierLine && (!hideOffscreenLine || !offScreen) && !isOverlapping;
         const opacity = isOverlapping && opacityOnOverlap;
+
         return (
             showBarrier && (
                 <div
                     className={`chart-line horizontal ${draggable ? 'draggable' : ''} ${isDragging ? 'dragging' : ''} ${className || ''}`}
-                    style={{ top: 0, color: foregroundColor, borderColor: color }}
+                    style={{ top: 0, color: foregroundColor, backgroundImage: `linear-gradient(to left, ${color} 90%, ${color}00` }}
```

Comment:
Don't we need space between `${color}` and `00` in `${color}00`?
```suggestion
                    style={{ top: 0, color: foregroundColor, backgroundImage: `linear-gradient(to left, ${color} 90%, ${color} 00` }}
```

Reply:
> Don't we need space between `${color}` and `00` in `${color}00`?

Do we need to add a space? The hex colors shouldn't have hexs right?

Reply:
In standard form of linear-gradient we have `<color> <percent>`  pattern. if that `00` is showing percentage it should have a space before it. is it a percentage or something else? and if those `00` are something else, what are they?

Reply:
@mahdiaryayi-fs I think its a weird combo of percentage and hex values. 00 is a hex representation of 0%. Let me confirm and see if its on purpose.

Reply:
@mahdiaryayi-fs We are using a combination of `<color> <percent>` and `<hex><hex_opacity>` because it provides the result that best matches the design provided to us. If I'm not mistaken, `${color} 90%` will provide solid color for `90%` of the line then `${color}00` will fade to 0 opacity for the last 10%.

Reply:
If you are sure about how it is working it is fine.